test/tests/ver-1kb/ver-1kb.i

```diff
@@ -0,0 +1,211 @@
+diffusivity = '${units ${fparse 4.31e-6 * exp(-2818/T)} m^2/s}'
+n_sorption = 1
+unit_scale = 1
+unit_scale_neighbor = 1
+
+[Mesh]
+  [generated]
+    type = GeneratedMeshGenerator
+    dim = 1
+    nx = '${fparse nb_segments}'
+    xmax = '${fparse long_total}'
+  []
+  [enclosure_1]
+    type = SubdomainBoundingBoxGenerator
+    input = generated
+    block_id = 1
+    bottom_left = '0 0 0'
+    top_right = '${fparse 1/3 * long_total} 0 0'
+  []
+  [enclosure_2]
+    type = SubdomainBoundingBoxGenerator
+    input = enclosure_1
+    block_id = 2
+    bottom_left = '${fparse 1/3 * long_total} 0 0'
+    top_right = '${fparse long_total} 0 0'
+  []
+  [interface]
+    type = SideSetsBetweenSubdomainsGenerator
+    input = enclosure_2
+    primary_block = 1
+    paired_block = 2
+    new_boundary = interface
+  []
+[]
+
+[Variables]
+  [concentration_enclosure_1]
+    block = 1
+    order = FIRST
+    family = LAGRANGE
+    initial_condition = '${fparse initial_concentration_1}'
+  []
+  [concentration_enclosure_2]
+    block = 2
+    order = FIRST
+    family = LAGRANGE
+    initial_condition = '${fparse initial_concentration_2}'
+  []
+[]
+
+[AuxVariables]
+  [pressure_enclosure_1]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 1
+    initial_condition = '${fparse initial_pressure_1}'
+  []
+  [pressure_enclosure_2]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 2
+    initial_condition = '${fparse initial_pressure_2}'
+  []
+[]
+
+[Kernels]
+  [diffusion_enclosure_1]
+    type = MatDiffusion
+    variable = concentration_enclosure_1
+    diffusivity = ${diffusivity}
+    block = '1'
+  []
+  [time_enclosure_1]
+    type = TimeDerivative
+    variable = concentration_enclosure_1
+    block = '1'
+  []
+  [diffusion_enclosure_2]
+    type = MatDiffusion
+    variable = concentration_enclosure_2
+    diffusivity = ${diffusivity}
+    block = '2'
+  []
+  [time_enclosure_2]
+    type = TimeDerivative
+    variable = concentration_enclosure_2
+    block = '2'
+  []
+[]
+
+[AuxKernels]
+  [pressure_enclosure_1]
+    type = ParsedAux
+    variable = pressure_enclosure_1
+    coupled_variables = 'concentration_enclosure_1'
+    expression = '${fparse R*T}*concentration_enclosure_1'
+    block = 1
+    execute_on = 'initial timestep_end'
+  []
+  [pressure_enclosure_2]
+    type = ParsedAux
+    variable = pressure_enclosure_2
+    coupled_variables = 'concentration_enclosure_2'
+    expression = '${fparse R*T}*concentration_enclosure_2'
+    block = 2
+    execute_on = 'initial timestep_end'
+  []
+[]
+
+[InterfaceKernels]
+  [interface_sorption]
+    type = InterfaceSorption
+    K0 = ${solubility}
+    Ea = 0
+    n_sorption = ${n_sorption}
+    diffusivity = ${diffusivity}
+    unit_scale = ${unit_scale}
+    unit_scale_neighbor = ${unit_scale_neighbor}
+    temperature = '${fparse T}'
+    variable = concentration_enclosure_1
+    neighbor_var = concentration_enclosure_2
+    sorption_penalty = 1e1
+    boundary = interface
```

Comment:
```suggestion
    temperature = ${temperature}
    variable = concentration_enclosure_2
    neighbor_var = concentration_enclosure_1
    sorption_penalty = 1e1
    boundary = Block2_Block1
```
I think you had it set up in reverse here, no? The concentration in enclosure 2 is set by the pressure in enclosure 1. 
I'm also updating the name of the temperature and of the interface.